scala/Charts/README.md

```diff
@@ -0,0 +1,12 @@
+# Example JFreeCharts Application (Scala)
+
+This is a sample application which makes a chart based on remote server response.
+The application uses the existing Databricks sample datasets.
+
```

Comment:
Always good to call out external libraries used and link to them. It's like giving credits.

Reply:
Good idea